apps/explorer/src/ui/Link.tsx

```diff
@@ -31,6 +32,7 @@ export interface LinkProps extends ButtonOrLinkProps, VariantProps<typeof linkSt
 	before?: ReactNode;
 	after?: ReactNode;
 	gap?: 'gap-1' | 'gap-2';
+	display?: 'inline-flex' | 'block' | 'flex';
```

Comment:
this should probably be added as a style variant instead of a prop

Reply:
style variants are only for the wrapper of the Link component. I will make an additional cva for this.